soc/arm64/bcm2712/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+zephyr_sources_ifdef(CONFIG_ARM_MMU mmu_regions.c)
+
+set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/include/zephyr/arch/arm64/scripts/linker.ld CACHE INTERNAL "")
```

Comment:
Could you add some comments describing why is this needed pls?

Reply:
same as above